types/dropzone/index.d.ts

```diff
@@ -299,8 +306,14 @@ declare class Dropzone {
 
 }
 
-interface JQuery {
-	dropzone(options: Dropzone.DropzoneOptions): Dropzone;
+declare global {
+	interface JQuery {
+		dropzone(options: Dropzone.DropzoneOptions): Dropzone;
+	}
+
+	interface HTMLElement {
```

Comment:
Could you point me to the place where HTMLElement is being augmented? I didn't found the documentation or example where this happens; otherwise I believe this is incorrect.

Reply:
The author of Dropzone has described this possibility when asked but has not added it to the documentation yet: https://github.com/enyo/dropzone/issues/180#issuecomment-19875928